corehq/apps/userreports/pillow.py

```diff
@@ -1,9 +1,13 @@
 from __future__ import absolute_import
+
+import pprint
 from collections import defaultdict
 from datetime import datetime, timedelta
 import hashlib
 
 from alembic.autogenerate.api import compare_metadata
+from django.conf import settings
+from django.core.mail import send_mail
```

Comment:
is there a reason you used this instead of the `send_mail_async` from `corehq.apps.hqwebapp.tasks`